marketplace/shared-dam-app/src/AppConfig/AppConfig.spec.tsx

```diff
@@ -39,23 +40,33 @@ const makeSdkMock = () => ({
   }
 });
 
+const validate = () => null; // Means no error
```

Comment:
Is this something that needs to be changed?

Reply:
it's a not problematic mock ;)